packages/fxa-auth-server/fxa-oauth-server/config/dev.json

```diff
@@ -86,7 +86,8 @@
       "imageUri": "https://example2.domain/logo",
       "redirectUri": "https://example2.domain/return?foo=bar",
       "trusted": true,
-      "canGrant": true
+      "canGrant": true,
+      "publicClient": true
```

Comment:
This will require a config update on stage/prod/testing.

Reply:
https://github.com/mozilla/fxa/blob/cf0435e4b20012e48f89234c0d99bdca3efef755/packages/fxa-auth-server/fxa-oauth-server/lib/routes/token.js#L382

Reply:
Past-me makes a compelling argument, 👍

Reply:
It's because we now call the /token endpoint rather than the /authorization endpoint to do direct grants. The token endpoint acts differently to the authorization endpoint in this respect. I could call the /code endpoint, but the way the auth-server's oauthdb is written, the endpoint requires a `state` token whereas /token does not. I could update these (and will open bugs), but we discussed using /token for direct grants instead of /authorization, and this seemed like the right way to go.

Reply:
> but we discussed using /token for direct grants instead of /authorization, and this seemed like the right way to go.

Yep, I recall this conversation and agree that using `/token` is the right way to go.